lib/contracts/code_generator.js

```diff
@@ -135,9 +135,9 @@ class CodeGenerator {
       } else {
         let connectionList = "[" + this.contractsConfig.dappConnection.map((x) => '"' + x + '"').join(',') + "]";
         if (self.env === 'development') {
-          web3Load = Templates.web3_connector({connectionList: connectionList, done: 'done();', warnAboutMetamask: true});
+          web3Load = Templates.web3_connector({connectionList: connectionList, done: 'done(err);', warnAboutMetamask: true});
         } else {
-          web3Load = Templates.web3_connector({connectionList: connectionList, done: 'done();', warnAboutMetamask: true});
+          web3Load = Templates.web3_connector({connectionList: connectionList, done: 'done(err);', warnAboutMetamask: true});
```

Comment:
hum.. looks like this if else is pointless ?, warnAboutMetamask shoudl be false on the else